src/i18n/translations/en.json

```diff
@@ -1,95 +1,85 @@
+  "report.monthly.civic-liker.v3.delegate.inactive": "Delegate now",
+  "report.monthly.civic-liker.v3.delegation-rewards": "New budget for reward this month",
+  "report.monthly.civic-liker.v3.disclaimer": "LIKE/USD price is calculated when sending this email",
+  "report.monthly.civic-liker.v3.distributerd-to": "Distributerd To",
+  "report.monthly.civic-liker.v3.top-supported": "Top Supported",
+  "report.monthly.civic-liker.v3.total-supported-balance": "Supported / Total Budget",
+  "report.monthly.civic-liker.v3.work": "{count, plural, one {Work} other {Works}}",
+  "report.monthly.creator.fund": "Creators Fund",
+  "report.monthly.creator.likers": "Likers",
+  "report.monthly.creator.likers.civic": "Civic Likers",
+  "report.monthly.creator.subject": "Monthly LikeCoin Reward Summary ({date})",
+  "report.monthly.creator.subscribers": "Monthly subscribers",
+  "report.monthly.creator.subscribers.legacy": "Other Civic Likers",
+  "report.monthly.creator.subscribers.legacy.hint": "(legacy)",
+  "report.monthly.creator.subscription": "Civic Liker Subscription",
+  "report.monthly.creator.title": "Monthly LikeCoin Reward Summary",
+  "report.monthly.creator.top-contents": "Your top works",
+  "report.monthly.creator.total": "Reward Received",
+  "report.monthly.creator.unit": "{value, plural, one {like} other {likes}}",
+  "report.monthly.creator.v2.top-contents": "Top Contents",
+  "report.monthly.creator.v2.top-supporters": "Top Supporters",
+  "report.monthly.creator.view-supporters": "View your supporters",
+  "report.monthly.date": "{month} {year}",
+  "resend": "Resend",
+  "reset-password.content": "Hi {name}, <br></br><br></br> We got a request to reset your Liker Land password. ",
+  "reset-password.cta": "Reset your password",
+  "reset-password.subject": "Reset Liker Land Password",
+  "reset-password.subtitle": "Reset Liker Land Password",
+  "reset-password.text": "Didn't make such request? <br></br>  Simply ignore this email and no changes were made to your account yet. <br></br> Please <a>contact us</a> if you need further support.",
+  "transaction.content": "Hi {name}, <br></br><br></br> You have received {amount} LikeCoin from {fromUser}. <br></br> Check the transaction details: ",
+  "transaction.remarks": "Remarks： {remarks}",
+  "transaction.subject": "You have received {amount} LikeCoin",
+  "transaction.subtitle": "You have received {amount} LikeCoin",
+  "unit.civic-likers": "{value, plural, one {Civic Liker} other {Civic Likers}}",
+  "unit.likers": "{value, plural, one {Liker} other {Likers}}",
+  "unit.likes": "{value, plural, one {Like} other {Likes}}",
+  "unit.works": "{value, plural, one {Work} other {Works}}",
+  "unsubscribe.text": "Unsubscribe",
+  "writing-nft_creator-follow_confirmation_content": "Please <a>click here</a> or the link below to confirm following {displayName}'s Writing NFT.",
+  "writing-nft_creator-follow_confirmation_subject": "Follow {displayName}'s Writing NFT",

```

Comment:
Any reason why only english is not in the format of `Writing NFT - Follow {displayName}` ?

Reply:
Not sure why, but seems like `Writing NFT - Follow {displayName}` is better for consistency

Reply:
Updated